src/block-editor/filters/withBlockBinding.tsx

```diff
@@ -102,12 +107,18 @@ export const withBlockBinding = createHigherOrderComponent( BlockEdit => {
 		const { remoteData } = useRemoteDataContext( context );
 		const availableBindings = getBlockAvailableBindings( remoteData?.blockName ?? '' );
 		const hasAvailableBindings = Boolean( Object.keys( availableBindings ).length );
+		const { hasMultiSelection } = useSelect< BlockEditorStoreSelectors >( blockEditorStore );
 
 		// If the block does not have a remote data context, render it as usual.
 		if ( ! remoteData || ! hasAvailableBindings ) {
 			return <BlockEdit { ...props } />;
 		}
 
+		// If multiple blocks are being selected, render it as usual.
+		if ( hasMultiSelection() ) {
```

Comment:
I think it might be how early this is positioned, that's causing this bug. It might be worth it to move into line 151 check? That has the updated attributes.

Reply:
Thanks, good spot. Fixed with test in https://github.com/Automattic/remote-data-blocks/pull/524/commits/eeab8c89861be29b1fb445875c8020015f6c2851

Reply:
Fix works 💯